### DISCUSSION TITLE
Merry Christmas 2024 & Happy New Year 2025

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#931] [Announcements]
Hi Astroiders,

As the sacred moment is approaching, I wish peace to all my friends. I am grateful to you for accompanying me this past year, witnessing Astroid grow and become stronger.

### Astroid Core

- Layout Builder UI/UX Upgrade
- Mega Menu Builder
- Dynamic Content
- Extra Field

### Astroid Pro

- Including 10+ Joomla Templates using only Astroid Framework
- 20+ Pro Widgets
- GSAP Animation on elements

As a thank you to you, I am happy to send you a gift of a 50% discount voucher for [all packages](https://astroidframe.work/pricing). It will be valid from now until 1/1/2025 -> https://astroidframe.work/pricing

![isonnyle_create_banner_for_christmas_event _Combination_of_aste_1ce1d77e-7843-4745-a515-c0cc39283fd9](https://github.com/user-attachments/assets/1d22aea3-a9f0-4bde-93e9-1d76b08fd7bd)

Thank you again and Merry Christmas.
With my best regards,
Sonny

**Comment by Dnsftl** [comment]
The best wishes and Merry Christmas to you.

Regards
Dennis

**Comment by srcarney** [comment]
I am looking forward to Astroid in 2025. In particular it says that you are adding Dynamic Content. That is something I really want and need. One feature I want to ask you to consider putting into your dynamic content is the ability to set an offset (a number value for the amount of articles that should be passed by) in order for me to make one row that features something like the most recent four items. Then I would create another row that might display the next 6 items. In order to do this I would want the offset on the second row to skip the first four items. etc...
Then I could possibly set up a third row with an offset of 10. This way there would be no duplicates. 

Stated another way:

Row 1:  4 Items. No offset
Row 2: 6 Items. Offset of 4
Row 3:  8 items. Offset of 10

These numbers are just for examples, not numbers to be built in. We need to be able to assign our offsets as we design. 

Thanks, Sean
